src/scripts/main.js

```diff
@@ -1,29 +1,65 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  // Creating HTML-element of notofication, adding classes and child elements
```

Comment:
There is a typo in the comment: 'notofication' should be corrected to 'notification'.